static/js/plotjs/foldphase.js

```diff
@@ -95,13 +94,28 @@ for (let i = 0; i < n_labels; i++) {
       blist.push({
         index: j,
         mjd_fold: foldb.data.mjd_foldb[j],
-        mag: foldb.data.mag[j],
+        mag: foldb.data.mag_unsmoothed[j],
       });
     }
     foldb.data.mag = sort_and_smooth(blist, binsize);
     for (j = 0; j < blist.length; j++) {
       foldberr.data.ys[j] = [foldb.data.mag[j], foldb.data.mag[j]];
     }
+  } else {
+    folda.data.mag = folda.data.mag_unsmoothed;
+    for (j = 0; j < folda.data.mag_unsmoothed.length; j++) {
+      foldaerr.data.ys[j] = [
+        folda.data.mag_unsmoothed[j],
+        folda.data.mag_unsmoothed[j],
+      ];
+    }
```

Comment:
Can reduce redundancy by setting the `foldaerr.data.ys` and `folberr.data.ys` values after the if/else statement is done. Regardless of smoothing or not, the 'ys' values can be set using the folda and foldb mag values, since those are being set within the if/else statement.

Reply:
@zvanderbosch Good point. fixed.